Source/Core/CesiumTerrainProvider.js

```diff
@@ -284,7 +285,7 @@ define([
                 if (defined(that._proxy)) {
                     metadataUrl = that._proxy.getURL(metadataUrl);
                 }
-                var parentMetadata = loadJson(metadataUrl);
+                var parentMetadata = loadJson(metadataUrl, undefined, undefined, requestOptions);
```

Comment:
You already know what I'm going to say: Forcing callers to pass in `undefined` as standard practice to skip arguments is bad API design.  Doesn't have to be this PR, but if this is going to go in largely as-is, we should submit an issue to update loadJson et al to take an options object sooner rather than later.